Nonprofit-Exchange-Hub/web-app
# fix dependencies being out of sync and vulnerabilities

from main run `npm install` in each directory and see that our package.lock was out of sync
so getting that up to date here
then i got the familiar prompt to run `npm audit fix`
```
jamesrogers@jamesrogpremiseMBP client % npm install
⸨#######⸨##################⸩ ⠸ reify:@testing-library/react: timing metavuln:cache:get:security-advisory:sockjs-client:Dbswe8TqDsFA3zYTVbBCvtrPW1tq8Y9a809OWa+uX9233MRpU4DJp
added 171 packages, removed 162 packages, changed 300 packages, and audited 1880 packages in 10s

1 package is looking for funding
  run `npm fund` for details

12 vulnerabilities (11 moderate, 1 high)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force
```
so i did!

to test:
1. `cd server && npm run start`
2. from another terminal`cd client && npm start`
3. http://localhost:3000/
4. click around and see nothing is broken
5. sign in and receive 200 response with access token
